### ISSUE TITLE
Buggy Scheduler in Conv Notebooks

### ISSUE THREAD

**Issue by IljaAvadiev** [World4AI/World4AI#27]
The notebooks that show basic cnn architectures use a scheduler with a mode="MAX", but use a validation loss. We have two options.

1. Use validation accuracy
2. Use MIN mode

Many of these notebooks will need to be rerun.

**Comment by IljaAvadiev** [comment]
The computer vision code was reorganized and the bugs were fixed.
